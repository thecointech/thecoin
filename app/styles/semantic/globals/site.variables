/*******************************
     User Global Variables
*******************************/

/*-------------------
       Fonts
--------------------*/

@fontName          : 'Muli';

/*-------------------
    Brand Colors
--------------------*/

@primaryColor        : #415DB0;
@secondaryColor      : #3193A1;

@complementaryColor	 : #FFC64A

@lightPrimaryColor   : #617AC5;
@lightSecondaryColor : #51ADBA;

@lightComplementaryColor: #FFDE98

/*-------------------
        Page
--------------------*/

@pageBackground      : #F0F0F0;

@lineHeight          : 1.6em;
@textColor           : rgba(0, 0, 0, 0.87);

