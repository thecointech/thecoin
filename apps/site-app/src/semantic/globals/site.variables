/*******************************
     Site-specific Global Variables
*******************************/

@pageBackground      : @theCoinPaletteGray2;
