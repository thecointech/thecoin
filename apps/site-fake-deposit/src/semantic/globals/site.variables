/*******************************
     Site-specific Global Variables
*******************************/

//@pageBackground      : @green;
