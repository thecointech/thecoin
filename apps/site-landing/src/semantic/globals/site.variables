/*******************************
     Site-specific Global Variables
*******************************/

//@pageBackground      : @green;

// A common definition for columns because we can't use a single grid for
// the entire page, so this provides single source for columns
@pageGridLeft: .scalable(20, 131)[@result];
@pageGridCenter: minmax(auto, 1178px);
@pageGridRight: .scalable(20, 131)[@result];
@pageGridContent: @pageGridLeft @pageGridCenter @pageGridRight;
