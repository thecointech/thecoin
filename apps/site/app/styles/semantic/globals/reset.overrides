/*******************************
         Site Overrides
*******************************/
*{
	margin: 0;
	padding: 0;
  border: none;
  box-shadow: none;
	font-size: 100%;
	font: inherit;
	vertical-align: baseline;
  background: transparent;
  color: inherit;
  padding: 0;
}

.ui.segments .segment, .ui.container .ui.segment, .ui.container{
  border: none;
  box-shadow: none;
  margin-top: 10px;
}

.ui.container {
  margin-left: 0 !important;
  margin-right: 0 !important;
}