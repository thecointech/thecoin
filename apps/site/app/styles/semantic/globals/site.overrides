/*******************************
         Site Overrides
*******************************/

a{
    font-size: 12px;
    font-weight: @bold;
    text-transform: uppercase;
    padding-bottom: 10px;
    border-bottom: 2px solid @linkUnderline;
}