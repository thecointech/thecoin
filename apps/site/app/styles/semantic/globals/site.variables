/*******************************
     User Global Variables
*******************************/

/*-------------------
        Page
--------------------*/

@pageBackground      : @white;
@maxContentWidth: 1440px;

/*-------------------
       Fonts
--------------------*/

@fontName2         : 'Abril Fatface';
@fontName          : 'Montserrat';

@headerFont        : @fontName2, 'Helvetica Neue', Arial, Helvetica, sans-serif;
@pageFont          : @fontName, 'Helvetica Neue', Arial, Helvetica, sans-serif;


/*-------------------
    Brand Colors
--------------------*/

@primaryColor        : #F18182;
@secondaryColor      : #20B58D;

/*--------------
  Page Heading
---------------*/

@headerFontWeight : @bold;
@headerLineHeight : unit((18 / 14), em);

@h1 : unit((80 / 14), rem);
@h2 : unit((64 / 14), rem);
@h3 : unit((40 / 14), rem);
@h4 : unit((28 / 14), rem);
@h5 : unit((14 / 14), rem);


/*-------------------
       Links
--------------------*/

@linkColor           : #61C1B8;
@linkUnderline       : @primaryColor;
@linkHoverColor      : darken(saturate(@linkColor, 20), 15, relative);
@linkHoverUnderline  : @linkUnderline;


/*-------------------
     Transitions
--------------------*/

@defaultDuration : 0.1s;
@defaultEasing   : ease;


/*-------------------
     Screen sizes
--------------------*/

@small: ~"(max-width: 699px) and (min-width: 520px)";
@medium: ~"(max-width: 1000px) and (min-width: 700px)";
@large: ~"(min-width: 1001px)";
@xlarge: ~"(min-width: 1151px)";