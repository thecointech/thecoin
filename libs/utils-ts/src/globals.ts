import { Firestore } from '@thecointech/types';

type TheCoinGlobals = {
  firestore: Firestore|null;
}
declare global {
  namespace globalThis {
    //  deepcode ignore no-var-keyword: var is necessary for this typing to work
    var __thecoin: TheCoinGlobals
  }
}
// initialize to empty
globalThis.__thecoin = {
  firestore: null,
}
