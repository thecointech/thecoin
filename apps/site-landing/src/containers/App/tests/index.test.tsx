import React from 'react';
import { jest } from '@jest/globals';
import rtr from 'react-test-renderer/shallow';

const renderer = rtr.createRenderer();

// Mock redux hooks
jest.unstable_mockModule('redux-injectors', () => ({
  useInjectSaga: jest.fn(),
  useInjectReducer: jest.fn(),
}));
// This import can trigger a compilation of the entire app
// (with numerous side-effects) so we skip it entirely
// NOTE: It only adds about 5 seconds to testing, should we keep it?
jest.unstable_mockModule('../Routes', () => ({
  Routes: () => <div>Router</div>
}));


const { App } = await import('../index');

describe('<App />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(<App />);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
