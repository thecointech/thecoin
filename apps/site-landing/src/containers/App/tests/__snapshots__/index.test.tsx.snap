// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render and match the snapshot 1`] = `
<MediaContextProvider>
  <style>
    .fresnel-container{margin:0;padding:0;}
@media not all and (min-width:0px) and (max-width:767px){.fresnel-at-mobile{display:none!important;}}
@media not all and (min-width:768px) and (max-width:1023px){.fresnel-at-tablet{display:none!important;}}
@media not all and (min-width:1024px) and (max-width:1191px){.fresnel-at-computer{display:none!important;}}
@media not all and (min-width:1192px) and (max-width:1919px){.fresnel-at-largeScreen{display:none!important;}}
@media not all and (min-width:1920px){.fresnel-at-widescreen{display:none!important;}}
@media not all and (max-width:767px){.fresnel-lessThan-tablet{display:none!important;}}
@media not all and (max-width:1023px){.fresnel-lessThan-computer{display:none!important;}}
@media not all and (max-width:1191px){.fresnel-lessThan-largeScreen{display:none!important;}}
@media not all and (max-width:1919px){.fresnel-lessThan-widescreen{display:none!important;}}
@media not all and (min-width:768px){.fresnel-greaterThan-mobile{display:none!important;}}
@media not all and (min-width:1024px){.fresnel-greaterThan-tablet{display:none!important;}}
@media not all and (min-width:1192px){.fresnel-greaterThan-computer{display:none!important;}}
@media not all and (min-width:1920px){.fresnel-greaterThan-largeScreen{display:none!important;}}
@media not all and (min-width:0px){.fresnel-greaterThanOrEqual-mobile{display:none!important;}}
@media not all and (min-width:768px){.fresnel-greaterThanOrEqual-tablet{display:none!important;}}
@media not all and (min-width:1024px){.fresnel-greaterThanOrEqual-computer{display:none!important;}}
@media not all and (min-width:1192px){.fresnel-greaterThanOrEqual-largeScreen{display:none!important;}}
@media not all and (min-width:1920px){.fresnel-greaterThanOrEqual-widescreen{display:none!important;}}
@media not all and (min-width:0px) and (max-width:767px){.fresnel-between-mobile-tablet{display:none!important;}}
@media not all and (min-width:0px) and (max-width:1023px){.fresnel-between-mobile-computer{display:none!important;}}
@media not all and (min-width:0px) and (max-width:1191px){.fresnel-between-mobile-largeScreen{display:none!important;}}
@media not all and (min-width:0px) and (max-width:1919px){.fresnel-between-mobile-widescreen{display:none!important;}}
@media not all and (min-width:768px) and (max-width:1023px){.fresnel-between-tablet-computer{display:none!important;}}
@media not all and (min-width:768px) and (max-width:1191px){.fresnel-between-tablet-largeScreen{display:none!important;}}
@media not all and (min-width:768px) and (max-width:1919px){.fresnel-between-tablet-widescreen{display:none!important;}}
@media not all and (min-width:1024px) and (max-width:1191px){.fresnel-between-computer-largeScreen{display:none!important;}}
@media not all and (min-width:1024px) and (max-width:1919px){.fresnel-between-computer-widescreen{display:none!important;}}
@media not all and (min-width:1192px) and (max-width:1919px){.fresnel-between-largeScreen-widescreen{display:none!important;}}
  </style>
  <div>
    <MainNavigation />
    <div>
      <MainPageTransition>
        <section>
          <Routes />
        </section>
      </MainPageTransition>
    </div>
    <Footer />
  </div>
</MediaContextProvider>
`;
